yt_dlp/extractor/facebook.py

```diff
@@ -500,6 +528,7 @@ def extract_metadata(webpage):
                 webpage, 'description', default=None)
             uploader_data = (
                 get_first(media, ('owner', {dict}))
+                or get_first(post, ('video', 'creation_story', 'attachments', ..., 'media', lambda k, v: k == 'owner' and v['name']))
```

Comment:
this also seems unrelated to any of the other changes in this PR, could be merged separately

Reply:
Would you like me to extract it to another PR?

Reply:
I've committed it to master and merged it into this PR